.github/ISSUE_TEMPLATE/new-plugin.md

```diff
@@ -0,0 +1,24 @@
+---
+name: New plugin
+about: Open Fixture Library should have a plugin for this fixture format.
+title: Add [software / console name] Plugin
+labels: new-plugin
+assignees: ''
+
+---
+
+XXXXX Please edit all lines that start with "XXXXX".  
+XXXXX Short description of the software / desk console using the fixture format.
```

Comment:
Please leave a newline here

```suggestion

XXXXX Short description of the software / desk console using the fixture format.
```

Reply:
Since the first two lines both need to be updated anyway, I'd leave them together in one paragraph. The first line has two trailing spaces to indicate a line break.